src/components/NFTBook/TippingDialog.vue

```diff
@@ -171,6 +171,7 @@ export default {
     },
   },
   mounted() {
+    logTrackerEvent(this, 'NFT', 'NFTTippingOpen', this.currentId, 1);
```

Comment:
is this a new event? probably use snake case for new event

Reply:
To be consistent with existing event in thos component, otherwise it is hard to filter